src/Dropdown/Dropdown.tsx

```diff
@@ -14,12 +14,14 @@ export interface DropdownProps {
   onChange: ChangeEventHandler;
   /** The name of the dropdown */
   name: string;
-  /** An array of objects with the properties 'label' and 'value' for each dropdown option */
-  options: Array<{label: string; value: string}>;
+  /** An array of objects with the properties 'label', 'value', and 'disabled' for each dropdown option */
+  options: Array<{label: string; value: string; disabled?: boolean}>;
   /** The currently selected dropdown value */
   value?: string;
   /** The dropdown value selected by default */
   defaultValue?: string;
+  /** If true, the dropdown is required to submit the form */
+  required?: boolean;
```

Comment:
This could also probably do with a test case

Reply:
Good point - I added a test case for this through 1fa513b.